data/sql/updates/pending_db_world/Citizen_of_Havenshire.sql

```diff
@@ -0,0 +1,772 @@
+(-128958, 0, 12, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 0, 232, 12895800, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Citizen of Havenshire - On Respawn - Start Path 12895800'),
+(-128958, 0, 13, 0, 109, 0, 100, 0, 0, 12895800, 0, 0, 0, 0, 41, 2000, 120, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Citizen of Havenshire - On Path 12895800 Finished - Despawn In 2000 ms');
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = -128910);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(-128910, 0, 12, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 0, 232, 12895800, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Citizen of Havenshire - On Respawn - Start Path 12895800'),
+(-128910, 0, 13, 0, 109, 0, 100, 0, 0, 12895800, 0, 0, 0, 0, 41, 2000, 120, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Citizen of Havenshire - On Path 12895800 Finished - Despawn In 2000 ms');
+
+-- Set Groups
+DELETE FROM `pool_template` WHERE (`entry` IN (22386, 22387, 22388, 22389, 22390, 22391, 22392, 22393, 22394, 22395, 22396, 22397, 22398, 22399, 22400, 22401, 22402, 22403, 22404, 22405, 22406, 22407));
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(22386, 1, 'Citizen of Havenshire random spawn 1'),
+(22387, 1, 'Citizen of Havenshire random spawn 2'),
+(22388, 1, 'Citizen of Havenshire random spawn 3'),
+(22389, 1, 'Citizen of Havenshire random spawn 4'),
+(22390, 1, 'Citizen of Havenshire random spawn 5'),
+(22391, 1, 'Citizen of Havenshire random spawn 6'),
+(22392, 1, 'Citizen of Havenshire random spawn 7'),
+(22393, 1, 'Citizen of Havenshire random spawn 8'),
+(22394, 1, 'Citizen of Havenshire random spawn 9'),
+(22395, 1, 'Citizen of Havenshire random spawn 10'),
+(22396, 1, 'Citizen of Havenshire random spawn 11'),
+(22397, 1, 'Citizen of Havenshire random spawn 12'),
+(22398, 1, 'Citizen of Havenshire random spawn 13'),
+(22399, 1, 'Citizen of Havenshire random spawn 14'),
+(22400, 1, 'Citizen of Havenshire random spawn 15'),
+(22401, 1, 'Citizen of Havenshire random spawn 16'),
+(22402, 1, 'Citizen of Havenshire random spawn 17'),
+(22403, 1, 'Citizen of Havenshire random spawn 18'),
+(22404, 1, 'Citizen of Havenshire random spawn 19'),
+(22405, 1, 'Citizen of Havenshire random spawn 20'),
+(22406, 1, 'Citizen of Havenshire random spawn 21'),
+(22407, 1, 'Citizen of Havenshire random spawn 22');
+
+DELETE FROM `pool_creature` WHERE (`pool_entry` IN (22386, 22387, 22388, 22389, 22390, 22391, 22392, 22393, 22394, 22395, 22396, 22397, 22398, 22399, 22400, 22401, 22402, 22403, 22404, 22405, 22406, 22407));
```

Comment:
and guid in (...)

Reply:
Is It really important? 
I used the pool_entry just to prevent to use guids. Currently It removes everything with those pool_entries (guids included).

Reply:
yes i would prefer having it. this way you can't accidently override existing entries with a pr.

Reply:
Solved.